transformation/ccda-to-fhirr4-service/tests/test_valid_ccda_document.xml

```diff
@@ -0,0 +1,342 @@
+<ClinicalDocument xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="urn:hl7-org:v3" xmlns:voc="urn:hl7-org:v3/voc" xmlns:sdtc="urn:hl7-org:sdtc">
```

Comment:
shall we add a simple test for the ccda document for which that heap exception occurred?

Reply:
As discussed offline, the specific scenario should be tested from the Ballerina standard libs.